mmv1/third_party/terraform/services/dataprocgdc/resource_dataproc_gdc_application_environment_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
The copyright will be added automatically and not needed.

Reply:
Done, removed the copyright